docs/howto/getting-started/accessing-cc-rest-api.md

```diff
@@ -0,0 +1,138 @@
+# Accessing the Cleura Cloud REST API
+
+{{brand}} provides a REST API, which you can take advantage of with a 
+tool like `curl`. But before you do, you must create a token for the 
+current session. For that, you only need to have an
+[account in {{brand}}](/howto/getting-started/create-account).
+
+## Creating a token
+
+To create a token, have your username (`your_cc_username`) and password 
+(`your_cc_password`) to the {{brand}} handy, and type:
+
+```bash
+curl -H "Content-Type: application/json" -X POST -d \
+    '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Provided you typed your username and password correctly and that there 
+were no connection issues to the remote endpoint, you will get a JSON 
+object with a string (`token`) that holds your session token 
+(`a_token_for_you`):
+
+```json
+{
+  "result": "login_ok",
+  "token": "a_token_for_you"
+}
+```
+
+## Testing the token
+
+One way to make sure the token is valid is by getting a list of all 
+supported regions. All you have to do is use `curl` to provide your 
+username (`your_cc_username`) and token (`a_token_for_you`) and connect 
+to the `https://rest.cleura.cloud/accesscontrol/v1/openstack/domains` 
+endpoint:
+
+```bash
+curl -H "X-AUTH-LOGIN: your_cc_username" -H "X-AUTH-TOKEN: a_token_for_you" \
+    https://rest.cleura.cloud/accesscontrol/v1/openstack/domains
+```
+
+All supported regions should be returned as objects in a JSON array:
+
+```JSON
```

Comment:
I *think* the syntax highlighter is case sensitive, and wants `json` in lowercase.

```suggestion
```json
```